assets/script.js

```diff
@@ -173,22 +171,58 @@ function filterbychr(input, chosenTag, tags) {
   getCommon(tagsL, chosenTag, input);
 }
 
-//need to clear storage at some point; ideally when clicking the back button (otherwise every quote will be saved and potentially appended to the page)
-function clearStorage() {
-  localStorage.clear();
-}
+//need to clear storage at some point
```

Comment:
we could probably just delete the clear storage stuff for now since yodaQuote fxn is always going to set most recent quote to local storage until we implement the ability to save multiple quotes